trackpy/diag.py

```diff
@@ -30,8 +31,12 @@ def dependencies():
         optional dependency is not installed, None
     """
     packages = ['six', 'numpy', 'scipy', 'matplotlib', 'pandas',
-                'scikit-image', 'pyyaml', 'pytables', 'numba', 'pyfftw']
+                'skimage', 'sklearn', 'pyyaml', 'tables', 'numba', 'pyfftw']
```

Comment:
I think pyfftw can be dropped now that we moved pyfftw to the `legacy_bandpass` function.

Also, the exception for pyfftw below can be cleaned up then

Reply:
Good call! I dropped pyfftw and added pims, since that seems to come up often when troubleshooting trackpy.